src/Client.js

```diff
@@ -66,6 +66,14 @@ export default class Client {
     }
   }
 
+  use(optionsUpdates) {
+    const options = {
```

Comment:
For consistency with the rest of the code, let's remove the trailing comma. We can also just shorten this to:

```js
return new this.contructor({ ...this.options, ...optionsUpdated });
```

Are we cool with this not being a deep merge? Any other authorized scopes that were pre-existing would be replaced, not merged.

Reply:
I think that's OK, but we should be explicit about it in the README